src/common-utils/main.sh

```diff
@@ -531,6 +545,7 @@ echo -e "\
     LOCALE_ALREADY_SET=${LOCALE_ALREADY_SET}\n\
     EXISTING_NON_ROOT_USER=${EXISTING_NON_ROOT_USER}\n\
     RC_SNIPPET_ALREADY_ADDED=${RC_SNIPPET_ALREADY_ADDED}\n\
-    ZSH_ALREADY_INSTALLED=${ZSH_ALREADY_INSTALLED}" > "${MARKER_FILE}"
+    ZSH_ALREADY_INSTALLED=${ZSH_ALREADY_INSTALLED}\n
+    MARKED_INSTALL_OH_MY_ZSH_CONFIG=${MARKED_INSTALL_OH_MY_ZSH_CONFIG}" > "${MARKER_FILE}"
```

Comment:
```suggestion
    INSTALL_OH_MY_ZSH_ALREADY_CONFIGURED=${INSTALL_OH_MY_ZSH_ALREADY_CONFIGURED}" > "${MARKER_FILE}"
```

Refactor to follow previous conventions?